src/utils/animation.ts

```diff
@@ -1,11 +1,13 @@
-import { SpringConfig, State, config } from "react-spring";
+import { SpringConfig, State } from "react-spring";
 
 export const animationConfig = {
   from: { opacity: 0, transform: "scale(0.95)" },
   enter: { opacity: 1, transform: "scale(1)" },
   leave: { opacity: 0, transform: "scale(0.95)", pointerEvents: "none" },
   config: ((_a: string, motion: State) =>
-    motion === "leave" ? { duration: 0.1 } : config.stiff) as (
-    key: string
-  ) => SpringConfig
+    motion === "leave"
+      ? { duration: 0.1 }
+      : {
+          duration: 200
+        }) as (key: string) => SpringConfig
```

Comment:
Write a comment that this `as` is because currently the types of react-spring are outdated and don't mention the second parameter.
`as` is similar to `ts-ignore` where you are sort of overriding typescript's inferred definitions.
So just write a comment as to why it is needed.